### PR TITLE
Fix link typo and reopen() clarification

### PR DESCRIPTION
**Pull request by adam-nielsen**
Change two [1] links to [1] and [2].  Clarify that reopen affects both past and future instances.

### CODE REVIEW

File: source/localizable/object-model/reopening-classes-and-instances.md [hunk 1]
```diff
@@ -29,16 +29,16 @@ across all instances of a class. It does not add
 methods and properties to a particular instance of a class as in vanilla JavaScript (without using prototype).
 
 But when you need to add static methods or static properties to the class itself
-you can use [`reopenClass()`][1].
+you can use [`reopenClass()`][2].
```
Comment (locks) [inline source/localizable/object-model/reopening-classes-and-instances.md]: Could you inline this instead? Thank you!

**Comment by adam-nielsen** [comment]
Updated as requested!

**Comment by locks** [comment]
👌  Thank you!
@homu +

**Comment by acorncom** [comment]
Looks like Homu is taking a break, merging manually

**Pull request merged**